example.env

```diff
@@ -0,0 +1,2 @@
+RECIPE_ENGINE=
+INPUT_VERSION=
```

Comment:
Is the current INPUT_VERSION set to the weird `20230426_nom2m3`? If so, I think we should archive that again so it follows the conventions

Reply:
see reply to data library PR [here](https://github.com/NYCPlanning/db-data-library/pull/392#issuecomment-1526275685) (sorry it's a paragraph)